vnctl/lib/vnctl/cli/base.rb

```diff
@@ -189,27 +179,28 @@ def rel_name
             desc "add #{base_uuid_label} #{relation_uuid_label} OPTIONS",
               "Adds #{desc_label} to a(n) #{parent.namespace}."
             def add(base_uuid, rel_uuid)
-              puts post("#{suffix}/#{base_uuid}/#{rel_name}/#{rel_uuid}", :query => options)
+              full_uri_suffix = "#{suffix}/#{base_uuid}/#{rel_name}/#{rel_uuid}"
+              puts Vnctl.webapi.post(full_uri_suffix, options)
```

Comment:
I think this reek is fine. `Vnctl.webapi` accesses a global variable holding a WebAPI object than can be used for sending HTTP requests.